src/validator/Validator.js

```diff
@@ -538,7 +538,16 @@ class ValidationChecker {
           });
         },
         check(component, setting, value) {
-          return (value !== 'Invalid date');
+          if (!value || value instanceof Date) {
```

Comment:
This looks good, although it has one problem: because JavaScript is weird, the following code is valid:
```
try {
  const myDate = new Date('hello, world!');
  console.log(myDate instanceof Date); // logs true
  console.log(typeof myDate === 'object'); // logs true
  console.log(myDate); // logs 'Invalid Date'
} catch (err) {
  // this code is unreachable, because the Date constructor will not throw an error
}
```

Because of this behavior, I think we need to move the `instanceof Date` check to after we check the `toString()` value to ensure that we're handling the case of `myDate instanceof Date === true && myDate.toString() === 'Invalid Date'`